linux/ibus-keyman/po/LINGUAS

```diff
@@ -0,0 +1,2 @@
+zh_CN
```

Comment:
Is this correct?

Reply:
> 
> 
> Is this correct?

Done.
that "translation" must have been left over from Doug copying from another ibus engine, so removed.